Changes

```diff
@@ -81,6 +81,11 @@ Working version
    which has never existed.
   (Jacques-Henri Jourdan, review by Xavier Leroy)
 
+- #1128, #9722: EINTR-based signals
```

Comment:
I would prefer a more descriptive entry, for example:
```
#1128, #7503, #9036, #9722: revised handling of signals.
Avoid running OCaml signal handlers in the middle of blocked system calls.
Instead, make sure system calls interrupted by a signal return quickly
to a program point where the signal handler can safely run.
Also revise the locking of I/O buffers to avoid deadlocks.
```

Reply:
+ a description of changes to the public API and polling behaviour.